deps/chakrashim/src/v8isolate.cc

```diff
@@ -136,7 +136,6 @@ void Isolate::SetCaptureStackTraceForUncaughtExceptions(
   //
   // kpathak: you might want to look into mechanism of captureStackTrace in
   // chakra_shim.js.
-  CHAKRA_ASSERT(false);
```

Comment:
I wanted to unblock execution and will circle back.  For non-critical functions I'm leaving the CHAKRA-TODO, but removing the assert.